src/xunit.v3.templates/templates/project/xunit3-extension/fs/SupportedOSAttribute.fs

```diff
@@ -37,16 +37,19 @@ type SupportedOSAttribute([<ParamArray>]supportedOSes: SupportedOS[]) =
 
         for supportedOS in supportedOSes do
             let osPlatform =
-                match osMappings.TryGetValue supportedOS with
-                | true, value -> value
-                | _ -> raise (new ArgumentException($"Supported OS value '{supportedOS}' is not a known OS", nameof(supportedOSes)))
+                match Map.tryFind supportedOS osMappings with
```

Comment:
very rare to see this unpiped, i.e. this is more common:
```
match osMappings |> Map.tryFind supportedOS with
```

Reply:
It's pretty common to see an unpiped `Map.tryFind` when used in a match statement: https://github.com/search?q=Map.tryFind+language%3AF%23&type=code&l=F%23

Reply:
It's a fair cop - I was using a _little_ poetic license to be fair!
(I am actually normally pretty anti egregious usage of piping, unless it truly makes the code more legible - I found it unintuitive as it stood, but that may be as a result of codebases I've worked in over the last while tending to using piping in such cases happening to arbitrarily standardise on that)
I wonder if Fantomas hasa rule/suggestion/preference on this [within the context of a `match`[?)